06-building-a-program-start-to-finish/delivering-your-product/README.md

```diff
@@ -0,0 +1,9 @@
+#### Output
+- **Using**: Medium
+- **Deliver**: a link to a post detailing your product and sharing your learning events/experiences
```

Comment:
What would you be looking for in such a blog post? Do we do something similar now with CPs?